### DISCUSSION TITLE
Pallets with logs from FS22_Forester_Economy are "stacked" across the trailer

### DISCUSSION THREAD

**Discussion by vitalii2011** [ddewar/FS22_UniversalAutoloadModhubAddon#31] [Non-Modhub Mod Support]
If there is time and interest, can you see what is wrong with them https://ls-modcompany.com/filebase/file/1123-forstwirtschaft-forestry-beta/ ? )
In UniversalAutoload.xml I set: 
```
    <containerConfigurations>
...
          <containerConfiguration name="FS22_Forester_Economy:woodPallet" containerType="EURO_PALLET" sizeX="4.0" sizeY="0.8" sizeZ="0.8"/>
    </containerConfigurations>
```

**Comment by ddewar** [comment]
Hey there @vitalii2011, try flipping your ```sizeX``` and ```sizeZ```. I also made ```sizeY="0.83"``` to be a little more exact. You can always use ```ctrl+shift+F12``` to get some debug boxes on to verify. 

Let me know if that helps!

```xml
<containerConfiguration name="FS22_Forester_Economy:woodPallet" containerType="EURO_PALLET" sizeX="0.8" sizeY="0.83" sizeZ="4.0"/>
```

**Comment by vitalii2011** [comment]
Thank you! Not very soon I would change the values of "sizeX and sizeZ". )

**Comment by vitalii2011** [comment]
Sorry, I could not figure out where it is correct to leave requests to add the mod "From ModHub". )

Lizard LT689 Long Log Trailer https://farming-simulator.com/mod.php?lang=en&country=gb&mod_id=245566 you can load 13 meter trees (cypresses) until pines and spruces have grown.

![fsScreen_2022_10_24_20_42_44](https://user-images.githubusercontent.com/41552153/198401042-4f17294e-5b44-4001-a86d-2e7be6e5273c.png)

**Reply by ddewar** [reply]
I'll work on getting this into the repo soon. I should add a few of the timber runners now that the main mod supports them.

**Reply by ddewar** [reply]
Check out the newest release for this trailer

https://github.com/ddewar/FS22_UniversalAutoloadModhubAddon/releases/tag/v1.0.0.3

**Comment by vitalii2011** [comment]
Thanks a lot!

**Comment by vitalii2011** [comment]
```
    <title>
        <en>Universal Autoload ModHub Add-on</en>
        <de>Universal Autoload ModHub-Zusatzmodul</de>
        <fr>Complément ModHub Pour Autoload Universel</fr>
        <ru>УА Для Техники С ModHub</ru>
        <nl>Universele Autoload ModHub Add-on</nl>
        <cz>Univerzální Autoload ModHub Doplněk</cz>
        <pl>Universalne Autoładowanie ModHub Add-on</pl>
    </title>
``` 
![fsScreen_2022_07_25_13_15_18](https://user-images.githubusercontent.com/41552153/202657581-864d80bb-0bd5-4994-a6c2-45fbe28b6e4b.png)
